tests/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.13-alpine
+
+# Install all needed tools
+RUN apk update && \
+  apk upgrade --update-cache --available && \
+  apk add build-base curl git
+
+# Install the executables for kubectl, rio, and hey
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+  chmod +x ./kubectl && \
+  mv ./kubectl /usr/local/bin && \
+  go get -u github.com/rakyll/hey
+
+# Set working directory to rio application directory and install all go dependencies
+WORKDIR /usr/local/projects/rio
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+# Copy source code into working directory and make sure entrypoint is executable
+COPY . .
+RUN chmod +x ./tests/entrypoint.sh
+
+# Set environment variables for tests to run properly and to ensure the correct rio version is being tested
+ENV INSTALL_RIO_VERSION=v0.6.0-alpha.2
```

Comment:
Could we get this use an ARG so we can `--build-arg` when running `docker build`? Or maybe move this to the `entrypoint.sh` script and set a default value for this variable.

Reply:
That's a good idea. It does the install in entrypoint, so setting this environment variable during a run does this behavior: `docker run -e INSTALL_RIO_VERSION=v0.6.0-alpha.x rio-test:latest`. Would it be better to set it during build or run do you think?

Reply:
No i think you're right, we will override it during `docker run`